<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="4" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-12T23:36:57 ICT" name="Default suite" finished-at="2022-01-12T23:37:38 ICT" duration-ms="41191">
    <groups>
    </groups>
    <test started-at="2022-01-12T23:36:57 ICT" name="Default test" finished-at="2022-01-12T23:37:38 ICT" duration-ms="41191">
      <class name="selenium.webdriver1.Xpath">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:selenium.webdriver1.Xpath@71809907]" started-at="2022-01-12T23:36:57 ICT" name="beforeClass" finished-at="2022-01-12T23:37:08 ICT" duration-ms="10784" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="TC_01_Register_Empty_Data()[pri:0, instance:selenium.webdriver1.Xpath@71809907]" started-at="2022-01-12T23:37:08 ICT" name="TC_01_Register_Empty_Data" finished-at="2022-01-12T23:37:17 ICT" duration-ms="9137" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_01_Register_Empty_Data -->
        <test-method signature="TC_02_Register_Invalid_Email()[pri:0, instance:selenium.webdriver1.Xpath@71809907]" started-at="2022-01-12T23:37:17 ICT" name="TC_02_Register_Invalid_Email" finished-at="2022-01-12T23:37:22 ICT" duration-ms="4747" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_02_Register_Invalid_Email -->
        <test-method signature="TC_03_Register_Incorrect_Email()[pri:0, instance:selenium.webdriver1.Xpath@71809907]" started-at="2022-01-12T23:37:22 ICT" name="TC_03_Register_Incorrect_Email" finished-at="2022-01-12T23:37:25 ICT" duration-ms="3360" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_03_Register_Incorrect_Email -->
        <test-method signature="TC_04_Register_Password_Less_Than_6_Chars()[pri:0, instance:selenium.webdriver1.Xpath@71809907]" started-at="2022-01-12T23:37:25 ICT" name="TC_04_Register_Password_Less_Than_6_Chars" finished-at="2022-01-12T23:37:29 ICT" duration-ms="3482" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_04_Register_Password_Less_Than_6_Chars -->
        <test-method signature="TC_05_Register_Incorrect_Password()[pri:0, instance:selenium.webdriver1.Xpath@71809907]" started-at="2022-01-12T23:37:29 ICT" name="TC_05_Register_Incorrect_Password" finished-at="2022-01-12T23:37:32 ICT" duration-ms="3340" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Mật khẩu nhập không khớp] but found [Mật khẩu phải có ít nhất 6 ký tự]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Mật khẩu nhập không khớp] but found [Mật khẩu phải có ít nhất 6 ký tự]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at selenium.webdriver1.Xpath.TC_05_Register_Incorrect_Password(Xpath.java:133)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_05_Register_Incorrect_Password -->
        <test-method signature="TC_06_Register_Invalid_Phone()[pri:0, instance:selenium.webdriver1.Xpath@71809907]" started-at="2022-01-12T23:37:32 ICT" name="TC_06_Register_Invalid_Phone" finished-at="2022-01-12T23:37:36 ICT" duration-ms="4470" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Số điện thoại không đúng: 012 - 02 -018 - 019] but found [Mật khẩu phải có ít nhất 6 ký tự]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Số điện thoại không đúng: 012 - 02 -018 - 019] but found [Mật khẩu phải có ít nhất 6 ký tự]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at selenium.webdriver1.Xpath.TC_06_Register_Invalid_Phone(Xpath.java:154)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_06_Register_Invalid_Phone -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:selenium.webdriver1.Xpath@71809907]" started-at="2022-01-12T23:37:36 ICT" name="afterClass" finished-at="2022-01-12T23:37:38 ICT" duration-ms="1798" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- selenium.webdriver1.Xpath -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
